resources/get-syn.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * Simple script that updates syn.js by downloading it from NPM through Unpkg and attaches a file
+ * header comment. To switch to a different Syn version, just update $version variable appropriately.
+ *
+ * @codeCoverageIgnore
+ */
+
+$version = '0.15.0';
+
+if ('cli' !== PHP_SAPI && 'phpdbg' !== PHP_SAPI) {
```

Comment:
I'd rather remove syn.js, its not maintained

Reply:
The selenium action api could definitely replace it. Of course we need to also consider older selenium as well.

At this point, if we decide to keep it, at least it's better to have this simple PHP script that explains/documents where syn came from.

I noticed you didn't take this approach, maybe you could try implementing your changes when this PR is merged (but before a first release)?

Reply:
Well, anything that can be done using the webdriver action API should be done with it. Places where Selenium2Driver is using syn.js are often places where it simulates half of the behavior but not all of it (for instance, the `focus` method simulates a focus event, but it might not actually focus the field).